efficientdet/backbone/efficientnet_model.py

```diff
@@ -219,53 +231,61 @@ def _build(self):
           kernel_initializer=conv_kernel_initializer,
           padding='same',
           data_format=self._data_format,
-          use_bias=False)
+          use_bias=False,
+          name=get_conv_name())
       self._bnsp = self._batch_norm(
           axis=self._channel_axis,
           momentum=self._batch_norm_momentum,
-          epsilon=self._batch_norm_epsilon)
+          epsilon=self._batch_norm_epsilon,
+          name=get_bn_name())
 
     filters = self._block_args.input_filters * self._block_args.expand_ratio
     kernel_size = self._block_args.kernel_size
 
-    # Fused expansion phase. Called if using fused convolutions.
-    self._fused_conv = self.conv_cls(
-        filters=filters,
-        kernel_size=[kernel_size, kernel_size],
-        strides=self._block_args.strides,
-        kernel_initializer=conv_kernel_initializer,
-        padding='same',
-        data_format=self._data_format,
-        use_bias=False)
-
-    # Expansion phase. Called if not using fused convolutions and expansion
-    # phase is necessary.
-    self._expand_conv = self.conv_cls(
-        filters=filters,
-        kernel_size=[1, 1],
-        strides=[1, 1],
-        kernel_initializer=conv_kernel_initializer,
-        padding='same',
-        data_format=self._data_format,
-        use_bias=False)
-    self._bn0 = self._batch_norm(
-        axis=self._channel_axis,
-        momentum=self._batch_norm_momentum,
-        epsilon=self._batch_norm_epsilon)
-
-    # Depth-wise convolution phase. Called if not using fused convolutions.
-    self._depthwise_conv = self.depthwise_conv_cls(
-        kernel_size=[kernel_size, kernel_size],
-        strides=self._block_args.strides,
-        depthwise_initializer=conv_kernel_initializer,
-        padding='same',
-        data_format=self._data_format,
-        use_bias=False)
+    if self._block_args.fused_conv:
+      # Fused expansion phase. Called if using fused convolutions.
+      self._fused_conv = self.conv_cls(
```

Comment:
@fsx950223 these changes are also for supporting model.summary.

Reply:
github did a bad job on diff (the actual changes are pretty simple: added if, else, and name=).

Reply:
I don't recommend use summary with subclass model.
It's design for keras graph model.